legislators-social-media.yaml

```diff
@@ -3478,6 +3480,7 @@
     thomas: '01654'
     govtrack: 400074
   social:
+    twitter: lacyclaymo1
```

Comment:
The [Twitter account](https://twitter.com/lacyclaymo1) says it's the "Official Campaign Twitter Account" and links to his campaign website, so this should not be included.